README.md

```diff
@@ -1,3 +1,5 @@
+# __IMPORTANT NOTE: READ-ONLY REPO, SCS2 GOT MOVED TO GITHUB: https://github.com/ihmcrobotics/simulation-construction-set-2 __
+
```

Comment:
What is this addition to the README?

Reply:
I think this was a Sylvain addition, may not be needed